spec.bs

```diff
@@ -1398,7 +1397,7 @@ To <dfn>fetch trusted signals</dfn> given a [=URL=] |url|, and a [=boolean=] |is
 <div algorithm>
 
 To <dfn>encode trusted signals keys</dfn> given an [=ordered set=] of [=strings=] |keys|:
-1. Let |list| be a new empty [=list=].

```

Comment:
I would keep the "empty". It's not a big deal, but it's a little less ambiguous.

Also try to avoid these miscellaneous changes in the middle of a PR review, just like you'd avoid them in the middle of a C++ code review. It's better to keep the big PRs focused, and make minor cleanups in their own PRs.